app/controllers/scheduler_controller.rb

```diff
@@ -393,8 +393,8 @@ def render_ical_event(section)
         endingMinutes = endTimeString.slice(endTimeString.size-2, 2)
 
         # Construct DateTime objects using the above values (and hardcoded august)
-        event_start = DateTime.new(2016, 1, eventDate, startingHour.to_i, startingMinutes.to_i, 1) #seconds have to be something otherwise it doesn't add? (weird af)
-        event_end = DateTime.new(2016, 1, eventDate, endingHour.to_i, endingMinutes.to_i, 1)
+        event_start = DateTime.new(2016, 8, eventDate, startingHour.to_i, startingMinutes.to_i, 1) #seconds have to be set otherwise it doesn't add
+        event_end = DateTime.new(2016, 8, eventDate, endingHour.to_i, endingMinutes.to_i, 1)
```

Comment:
Want to go ahead and take this number out and save it as the month (and maybe year while we're at it)? That should make it easier to set if/when we scrape a page for the info.